packages/teleport/src/Databases/AddDatabase/Manually/Manually.tsx

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright 2021 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import { Text, Box, ButtonSecondary, Link } from 'design';
+import { DialogContent, DialogFooter } from 'design/Dialog';
+import Select, { Option } from 'shared/components/Select';
+import cfg from 'teleport/config';
+import TextSelectCopy from 'teleport/components/TextSelectCopy';
+import * as links from 'teleport/services/links';
+import { DbType, DbProtocol } from 'teleport/services/databases';
+import { formatDatabaseInfo } from 'teleport/services/databases/makeDatabase';
+
+export default function Manually({ user, version, onClose }) {
+  const { hostname, port } = window.document.location;
+  const host = `${hostname}:${port || '443'}`;
+
+  const [dbOptions] = useState<Option<Database>[]>(() =>
+    options.map(dbOption => {
+      return {
+        value: dbOption,
+        label: dbOption.title,
+      };
+    })
+  );
+
+  const [selectedDbOption, setSelectedDpOption] = useState<Option<Database>>(
+    dbOptions[0]
+  );
+
+  return (
+    <>
+      <DialogContent minHeight="240px" flex="0 0 auto">
```

Comment:
what does `minHeight` and `flex` do here?

Reply:
Nothing :sweat_smile: